tests/pytests/test_hayabusa_drivers.py

```diff
@@ -0,0 +1,92 @@
+import os
+import json
+import unittest
+from unittest.mock import patch
+
+import pytest
+import numpy as np
+import spiceypy as spice
+
+import ale
+from ale.drivers.hayabusa_drivers import HayabusaAmicaIsisLabelNaifSpiceDriver
+from ale.formatters.isis_formatter import to_isis
+
+from conftest import get_image_kernels, convert_kernels, get_image_label, compare_dicts
+
+@pytest.fixture()
+def isis_compare_dict():
+    return {
+        'CameraVersion': 1,
+        'NaifKeywords': {
+            'BODY_CODE' : 2025143,
+            'BODY2025143_RADII' : [0.274, 0.156, 0.138],
+            'BODY_FRAME_CODE' : 2025143,
+            'INS-130102_SWAP_OBSERVER_TARGET' : 'TRUE',
+            'INS-130102_LIGHTTIME_CORRECTION' : 'NONE',
+            'INS-130102_LT_SURFACE_CORRECT' : 'FALSE',
+            'INS-130102_FOCAL_LENGTH' : 0.1208,
+            'INS-130102_PIXEL_PITCH' : 0.012,
+            'CLOCK_ET_-130_2385525447_COMPUTED' : '1c570a3ae87ca541',
+            'INS-130102_TRANSX' : [0.0, 0.012, 0.0],
+            'INS-130102_TRANSY' : [0.0, 0.0, -0.012],
+            'INS-130102_ITRANSS' : [0.0, 83.33333333, 0.0],
+            'INS-130102_ITRANSL' : [0.0, 0.0, -83.33333333],
+            'INS-130102_BORESIGHT_LINE' : 511.5,
+            'INS-130102_BORESIGHT_SAMPLE' : 511.5,
+            'INS-130102_OD_K' : [0.0, 2.8e-05, 0.0]
+        },
+        'InstrumentPointing': {
+            'TimeDependentFrames' : [-130000, 1],
+            'ConstantFrames' : [-130102, -130101, -130000],
```

Comment:
Until we get the frame chain fully implemented, the to_isis formatter just returns this as 1 step.